packages/main/src/Input.ts

```diff
@@ -1491,7 +1491,7 @@ class Input extends UI5Element implements SuggestionComponent, IFormElement {
 	}
 
 	announceSelectedItem() {
-		const invisibleText = this.shadowRoot!.querySelector(`#${this._id}-selectionText`)!;
+		const invisibleText = this.shadowRoot!.getElementById(`#${this._id}-selectionText`)!;
```

Comment:
Can we use similar solution like #7519 by simply use $0.querySelector(`[id="some-id"]`) for consistency?

Reply:
I've updated all the code to ```querySelector(`[id="xxx"]`)```. But I'm not sure if it's a good idea since some character like double-quotes is still not supported.
BTW the unit test fails, I have no idea what happened, will check it later